src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -461,6 +497,27 @@ protected void updateFallState(double y, boolean onGroundIn, IBlockState state,
         super.updateFallState(y, onGroundIn, state, pos);
     }
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'sourceEntity' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)